exercises/zebra-puzzle/.meta/template.j2

```diff
@@ -0,0 +1,12 @@
+{%- import "generator_macros.j2" as macros with context -%}
+{{ macros.header() }}
+
+class {{ exercise | camel_case }}Test(unittest.TestCase):
+    {% for case in cases -%}
+    def test_{{ case["description"] | to_snake }}(self):
+        expected = {{ case["expected"] }}
+        self.assertEqual({{ case["property"] }}(), expected)
```

Comment:
How about:
```
self.assertEqual({{ case["property"] }}(), {{ case["expected"] }})
```

Reply:
I changed before I saw this.

```
expected = "{{ case["expected"] }}"
'''

is this okay?

Reply:
I think it'd still be nice to get this on one line in the generated output, like in the original. I'm mass-reviewing PRs for test templates, I'm much less likely to miss something if the diff is smaller.

Reply:
That makes sense. I think everything is good here now. Thanks for the help!